File: Python3/프로그래머스/1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.py
def solution(s):
    count = 0
    
    for ch in s:
        if ch == 'p' or ch == 'P':
            count += 1
        if ch == 'y' or ch == 'Y':
            count -= 1
    
    if count == 0:
        print("True")
        return True
    else:
        print("False")
        return False

File: C++/프로그래머스/1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.cpp
#include <string>
#include <iostream>
using namespace std;

bool solution(string s) {
    int count = 0;

    for (char ch : s) {
        if (ch == 'p' || ch == 'P') count++;
        if (ch == 'y' || ch == 'Y') count--;
    }

    if (count == 0) {
        cout << "True" << endl;
        return true;
    } else {
        cout << "False" << endl;
        return false;
    }
}

int main() {
    string input = "pPoooyY";
    solution(input);
    return 0;
}
